ingestion/src/metadata/profiler/interface/sqlalchemy/bigquery/profiler_interface.py

```diff
@@ -32,6 +32,9 @@ def _get_struct_columns(self, columns: dict, parent: str):
         for key, value in columns.items():
             if not isinstance(value, STRUCT):
                 col = Column(f"{parent}.{key}", value)
+                # pylint: disable=protected-access
+                col._set_parent(self.table.__table__)
```

Comment:
```suggestion
                col._set_parent(self.table.__table__)  # pylint: disable=protected-access
```

Would this be the same? or does it get too long

Reply:
It gets too long